.gitignore

```diff
@@ -146,3 +146,9 @@ dist
 
 # Locally generated PII reports
 pii_report
+
+# transifex configurations
+.transifexrc
+
+# adg edx-theme
+themes/adg-edx-theme/
```

Comment:
Can we make theme repo as a git [Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules), instead of adding it to gitignore

Reply:
Leaving it for now.